schemas/triniti/canvas/mixin/iframe-block/latest.xml

```diff
@@ -10,6 +10,9 @@
       <field name="width" type="string" pattern="^\d+(%|px)?$"/>
       <field name="height" type="string" pattern="^\d+(%|px)?$"/>
       <field name="align" type="string" pattern="^(center|right|left)$"/>
+      <field name="scrolling_enabled" type="boolean">
```

Comment:
@nanetteCodes - please add a description tag like this: https://github.com/triniti/schemas/commit/1c019bb647531123b81b6291b5476ef696e04c6f#diff-57aa758b039e616d5bf075c0478d47dbR72